package.json

```diff
@@ -81,6 +81,11 @@
       "types": "./dist/window.d.ts",
       "import": "./dist/window.js",
       "require": "./dist/window.cjs"
+    },
+    "./starknet-react": {
```

Comment:
should be starknetkit-react-native, otherwise it would be confused for https://www.starknet-react.com/docs/getting-started

same for the folder name under /src